pom.xml

```
@@ -1778,6 +1778,9 @@
                             -Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager
                             -Daws.region=us-east-1 <!-- required for s3-related unit tests -->
                             -Ddruid.test.stupidPool.poison=true
+                            <!-- the path needs to come from a maven option or so -->
+                            -XX:OnOutOfMemoryError='chmod 644 /home/runner/work/druid/druid/*.hprof'
```

Will environment variables be respected here?
```suggestion
                            -XX:OnOutOfMemoryError='chmod 644 ${GITHUB_WORKSPACE}/*.hprof'
```

---

Or maybe it can be path relative so this works everywhere, not just in Github Actions?

---

Yeah, I'm not sure how best to do this. Ideally it'd be some relative path like you're referring to

---

Updated to use `${project.parent.basedir}/target`

---

please use `${maven.multiModuleProjectDirectory}` ; I think that property will not necessarily get to the project root